gnmidiff/json.go

```diff
@@ -123,7 +127,12 @@ func flattenOCJSONAux(root interface{}, path string, leaves map[string]interface
 	case map[string]interface{}:
 		// This is a container or a list element.
 		for subpath, subv := range v {
-			if err := flattenOCJSONAux(subv, path+"/"+subpath, leaves); err != nil {
+			if !keepNamespace {
+				if idx := strings.IndexRune(subpath, ':'); idx != -1 {
```

Comment:
I'm not sure which is more efficient, but this seemed a bit more logical to me -- completely aesthetic though, so feel free to ignore this.

```golang
pp := strings.Split(subpath, ":")
subpath = pp[len(pp)-1]
```

Reply:
Nice one, updated. yes I was aiming for some aesthetics here.